### PR TITLE
9.0-mig-web_ir_actions_act_window_none

### PR DESCRIPTION
**Pull request by susport**
Mig 9.0 web_ir_actions_act_window_none

### CODE REVIEW

**Comment by hbrunn** [comment]
this works just like that because of the compatibility wrapper. but you should migrate this to the `require` api. which will also be a better argument to add yourself to the list of contributors.

and would you care to add js tests?

File: web_ir_actions_act_window_none/static/src/js/web_ir_actions_act_window_none.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+//-*- coding: utf-8 -*-
+//?? 2016 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+odoo.web_ir_actions_act_window_none = function(instance)
```
Comment (hbrunn) [inline web_ir_actions_act_window_none/static/src/js/web_ir_actions_act_window_none.js]: this is not yet v9 api. https://www.odoo.com/documentation/9.0/reference/javascript.html#web-client should give you the idea

File: web_ir_actions_act_window_none/README.rst [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+    :target: https://runbot.odoo-community.org/runbot/162/9.0
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/web/issues>`_.
```
Comment (tarteo) [inline web_ir_actions_act_window_none/README.rst:35]: Please update this section to the latest version of the template.

File: web_ir_actions_act_window_none/static/src/js/web_ir_actions_act_window_none.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+//-*- coding: utf-8 -*-
+//?? 2016 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
+odoo.define('web_ir_actions_act_window_none', function(require) {
+
```
Comment (tarteo) [inline web_ir_actions_act_window_none/static/src/js/web_ir_actions_act_window_none.js:7]: "use strict"; is for all javascript files recommended.

**Review by tarteo — APPROVED** [review]
I only have some nitpicking. 👍

File: web_ir_actions_act_window_none/README.rst [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+Usage
+=====
+
+To use this module, depend on it in your module and return::
+
+    {'type': 'ir.actions.act_window.none'}
+
+The button will trigger a reload of the popup's form, in case you want to
+suppress this, set key ``reload`` to ``False`` in the action you return.
```
Comment (yajo) [inline web_ir_actions_act_window_none/README.rst:22]: Assuming you set a `none` action with `reload=False`...... What's this for? An addon that lets buttons do nothing?

You should drop support of `reload=False` and call this `web_action_reload`, I'm not in favor of merging an addon that adds a feature that does nothing, honestly 😕.

**Review by yajo — CHANGES_REQUESTED** [review]

**Pull request closed without merging**